src/main/webapp/sharing/user-share-panel.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import FormControl from '@material-ui/core/FormControl'
+import Button from '@material-ui/core/Button'
+import UserAddRow from './user-add-row'
+
+export default props => {
+  const { state, handleChange, removeUser, addUser } = props
+
+  return (
+    <FormControl style={{ width: '100%' }}>
+      <div style={{ maxHeight: '400px', overflow: 'auto', paddingTop: '.5em' }}>
```

Comment:
:pencil2: we use `em` in some places and `rem` in others for sizing. it would be nice to be consistent